bankroll/converter/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .modelconverter import ModelConverter
+
+# __all__ = ["ModelConverter"]
```

Comment:
Did you mean to comment this out?

Reply:
✅ Now using the converter package properly